src/weather.js

```diff
@@ -39,6 +39,13 @@ function updateToday(dataObject) {
   const temperature = document.querySelector(".today .temperature");
   const minTemp = document.querySelector(".today .min-temp");
   const maxTemp = document.querySelector(".today .max-temp");
+
+  city.textContent = dataObject.location.name;
+  img.src = "http:" + dataObject.current.condition.icon;
```

Comment:
I think the "http:" at the start is unnecessary.
Dataobject.current.condition.icon should return the full url.

Reply:
> I think the "http:" at the start is unnecessary. Dataobject.current.condition.icon should return the full url.

Adding the "http:" makes it an absolute URL, I think this is a good addition.